test/core/test/exports.test.ts

```diff
@@ -0,0 +1,205 @@
+import { resolve } from 'node:path'
+import { expect, it } from 'vitest'
+import { getPackageExportsManifest } from 'vitest-package-exports'
+
+it('exports snapshot', async () => {
+  const manifest = await getPackageExportsManifest({
+    importMode: 'package', // or 'dist' or 'package'
+    cwd: resolve(import.meta.dirname, '../../../packages/vitest'),
+  })
+
+  expect(manifest.exports)
+    .toMatchInlineSnapshot(`
+      {
+        ".": {
+          "afterAll": "function",
+          "afterEach": "function",
+          "assert": "function",
+          "assertType": "function",
+          "beforeAll": "function",
+          "beforeEach": "function",
+          "bench": "function",
+          "chai": "object",
+          "createExpect": "function",
+          "describe": "function",
+          "expect": "function",
+          "expectTypeOf": "function",
+          "getRunningMode": "function",
```

Comment:
this one is not documented, alongside `runOnce`, `isFirstRun` and `isWatchMode` - should we get rid of them?

Reply:
let's remove it